sway/commands/workspace.c

```diff
@@ -181,7 +181,12 @@ struct cmd_results *cmd_workspace(int argc, char **argv) {
 			++argv;
 		}
 
-
+		bool create = false;
```

Comment:
This could be simplified to the following, which does fit on a single line:
```c
bool create = argc > 1 && strcasecmp(argv[1], "--create") == 0;
```